tdrs-frontend/src/components/Header/Header.jsx

```diff
@@ -57,24 +57,30 @@ function HeaderComp() {
               <div className="usa-nav__secondary">
                 <ul className="usa-nav__secondary-links">
                   <li className="usa-nav__secondary-item">
-                    <a href="/">
-                      <FontAwesomeIcon
-                        className="margin-right-1"
-                        icon={faUserCircle}
-                      />
-                      {user.email}
-                    </a>
+                    {user && user.email && (
+                      <a href="/">
+                        <FontAwesomeIcon
+                          className="margin-right-1"
+                          icon={faUserCircle}
+                        />
+                        {user && user.email}
+                      </a>
+                    )}
                   </li>
                   <li className="usa-nav__secondary-item">
                     <a
                       className="sign-out-link"
-                      href={`${process.env.REACT_APP_BACKEND_URL}/logout/oidc`}
+                      href={
+                        user && user.email
+                          ? `${process.env.REACT_APP_BACKEND_URL}/logout/oidc`
+                          : `${process.env.REACT_APP_BACKEND_URL}/login/oidc`
```

Comment:
This way the button will direct the user to the right place depending on their authentication.